### PR TITLE
Add deb package creation, upload to fury.io and Ubuntu apt installation documentation (TOOLS-84)

### PR DESCRIPTION
**Pull request by kobrikx**
#### What's new:

 - Added deb package of ize creation to GoReleaser workflow
 - Added upload of deb package to Gemfury.io repo (https://apt.fury.io/hazelops/)
 - Added instruction of how to install ize on linux using public repo

#### Testing done:

Installation from repo:

screenshot 2021-11-04 at 18 34 26

Upgrade from repo:

screenshot 2021-11-04 at 19 24 36

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -40,3 +40,33 @@ brews:
...
+    # You can change the file name of the package.
+    # Default: `{{ .PackageName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}`
+    file_name_template: "{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}"
+    formats:
+      - deb
+    vendor: HazelOps
+    homepage: https://ize.sh
+    maintainer: Ize development team <ize@hazelops.com>
+    description: IZE is an opinionated infrastructure wrapper that allows to use multiple tools in one infra
+    license: MIT
```
Comment (AutomationD) [inline .goreleaser.yml:63]: The license is specified in LICENSE (Apache 2.0, not MIT)

**Review by AutomationD — APPROVED** [review]
Looks great. Let's just fix the license.

File: README.md [hunk 2]
```diff
@@ -59,3 +59,35 @@ ize destroy infra
   4.2 Update version in brew repo: `brew tap hazelops/ize`
   4.3 Install `ize`: `brew install ize`
 
+## Ize installation via public apt repository URL (Ubuntu):
+
+##### 1. To enable, create the following file:
+
+ `/etc/apt/sources.list.d/fury.list` with the following content:
```
Comment (AutomationD) [inline README.md:66]: Let's make it as 
```
sudo echo "deb [trusted=yes] https://apt.fury.io/hazelops/ /" > /etc/apt/sources.list.d/fury.list
```

**Review by AutomationD — APPROVED** [review]
Looks good with some changes

**Review by AutomationD — APPROVED** [review]
Looks great. Let's just fix the license.

**Review by psihachina — APPROVED** [review]

**Pull request merged**